runtime/jcl/uma/sun_misc_Unsafe_exports.xml

```diff
@@ -37,7 +37,9 @@
 		<export name="Java_sun_misc_Unsafe_throwException" />
 		<export name="Java_sun_misc_Unsafe_tryMonitorEnter" />
 		<export name="Java_sun_misc_Unsafe_unpark" />
-		<export name="Java_sun_misc_Unsafe_defineAnonymousClass" />
+		<export name="Java_sun_misc_Unsafe_defineAnonymousClass">
+			<exclude-if condition="spec.java17" />
```

Comment:
Is `spec.java17` going to be set for future versions of java?

Reply:
Yes, `spec.java17` is equivalent to `JAVA_SPEC_VERSION >= 17`.